tests/features/change_tracking_workflow.feature

```diff
@@ -0,0 +1,170 @@
+      | <p>No ice-ins or ice-del tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p> | <p>No ice-ins or ice-del tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p> |
+      | <p>No tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p>                    | <p>No tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p>                    |
+
+  @api
+  Scenario Outline: The change of the content state to "validated" or "published" must be blocked if
+  CKEditor Lite tracked changes exist in WYSIWYG fields of a translation
+    Given I am logged in as a user with the 'administrator' role
+    And the following languages are available:
+      | languages |
+      | en        |
+      | fr        |
+    When I go to "admin/config/content/multisite_wysiwyg/workbench"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-validated"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-published"
+    And I press "Save configuration"
+    Then I should see the success message "The configuration options have been saved."
+    And I am viewing a multilingual "page" content:
+      | language | title              | body                 |
+      | en       | Title in English   | <p>Page body</p>     |
+      | fr       | Titre en Français  | <p>Corps de page</p> |
+    And I click "English" in the "header_top" region
+    And I click "Français"
+    And I click "New draft"
+    And I select "Full HTML + Change tracking" from "field_ne_body[fr][0][format]"
+    And I fill in "Body" with "<blocked>"
+    And I press "Save"
+    Then I should see the success message "Basic page Titre en Français has been updated."
+    When I click "Français" in the "header_top" region
+    And I click "English"
+    When I select "Published" from "state"
+    And I press "Apply"
+    Then I should see the error message "Tracked changes detected in the French version. To save progress, please accept or reject them, or change the content state."
+    When I select "Validated" from "state"
+    And I press "Apply"
+    Then I should see the error message "Tracked changes detected in the French version. To save progress, please accept or reject them, or change the content state."
+    When I select "Needs Review" from "state"
+    And I press "Apply"
+    Then I should not see the error message "Tracked changes detected in the French version. To save progress, please accept or reject them, or change the content state."
+
+    Examples:
+      | blocked                                                                                                                                                                                                                                  |
```

Comment:
I take into account the remark... but for the next time. On this ticket, I already spent a lot of time and I fear that kind of improvements consume some time in trial and error steps before having the perfect display.